src/main/webapp/js/grapheditor/Shapes.js

```diff
@@ -3646,6 +3646,21 @@ function getShape(cell) {
     return permission;
   }
 
+  function onCRUDActionValue(type) {
+    makeTitle = (type) => {
+      if (type === "create") {
+        return "Create";
+      } else if (type === "read") {
+        return "Read";
+      } else if (type === "update") {
+        return "Update";
+      } else if (type === "delete") {
+        return "Delete";
+      }
+    }
+    return (`On ${makeTitle(type)} Action`);
```

Comment:
You can simplify this as well:

```suggestion
      const title = {
        "create": "Create",
        "read": "Read",
        "update": "Update",
        "delete": "Delete",
      }[type];
      return (`On ${title} Action`);
```

Reply:
Also, since it is dupplicated and does the same thing as before, maybe turning it into a function?

I mean, DrawIO source-code already has a lot of dupplicates, I'm not sure we wanna be part of the problem 👀

Reply:
sure, where should I put it?